src/display.rs

```diff
@@ -288,6 +291,12 @@ impl Display {
         let size_info = *terminal.size_info();
         let visual_bell_intensity = terminal.visual_bell.intensity();
 
+        let background_color = terminal.get_background_color();
+        let background_color_changed = terminal.background_color_changed;
```

Comment:
I think it might make more sense to store `last_background_color` on the `Display` rather than the `Term`. The fact that it needs to be tracked at all is really due to a driver issue relating to the `Renderer`.